### PR TITLE
Fix typos and change formatting of README

### PR DESCRIPTION
**Pull request by johannaengland**
To get started with my general review, I did what I like to do first: read the documentation and without knowing the code reviewing that. Some typo fixes, the rest is generally just my preference when it comes to formatting etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,14 +20,14 @@ will result in the following diff:
...
 
 If there is no diff (e.g.: `diff({ "a": 1 }, { "a": 1 })`) the result will be `None`. In this sense,
 the library can also be used to compare two json-compatible dictionaries.
 
 ---
 
-If a object of any different type is used anywhere in a dict/list/..., a `ValidationException` will be raised.
+If an object of any other type is used in a dict, list, ..., a `ValidationException` will be raised.
 
 ## Exports
```
Comment (johannaengland) [inline README.md:32]: For this part I don't fully understand why it is part of the README?

File: README.md [hunk 2]
```diff
@@ -53,12 +53,13 @@ diff("a", "a") is None
 
 ---
 
-The diff will treat lists, sets and tuples as lists which means that the result will contain lists instead of sets or tuples.
+The diff will treat lists, sets and tuples as lists, which means that the result will contain lists instead of sets or tuples.
```
Comment (johannaengland) [inline README.md:56]: I would also like to have an example for this, because to me this sentence alone doesn't fully convey what you mean.

**Review by g3rrit — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (g3rrit) [reply README.md:32]: I think it doesn't hurt.

File: README.md (hunk 2 above)
Comment (g3rrit) [reply README.md:56]: Opened #5 for this :)

**Pull request merged**